.travis.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   include:
     - python: 3.6
     - python: 3.7
+    - python: 3.8
```

Comment:
was this change supposed to be part of the PR?

Reply:
@bdpedigo no but i am unsure of how to get rid of it. I even made a new branch.

Reply:
need to remove this commit https://github.com/microsoft/graspologic/pull/483/commits/757e483934aff17c45952de80b9824d168a26b2e from this branch

Reply:
@bdpedigo I am still unsure what i should do. I did git reset --hard 757e483. But it still appears in the file .travis.yml file. Should I have done one more commit before that?

Reply:
@dfrancisco1998 part of sprint 0 is typically banging your head against the github wall for at least a little bit... i still `git` frustrated by these things too. mind posting your question in the slack discussion channel for everyone to see and try to help?

Reply:
https://stackoverflow.com/a/40926981